manifests/charts/istio-cni/values.yaml

```diff
@@ -4,7 +4,9 @@ cni:
   image: install-cni
   pullPolicy: ""
 
-  logLevel: info
+  # Configuration log level of istio-cni binary
+  # by default istio-cni send all logs to UDS server
+  logLevel: debug
```

Comment:
so we send debug logs, but then to see them you need `logging.level=cni:debug`, right?

Can we explain that in the comments maybe?

Reply:
updated.